lib/Conversion/TorchToTcp/TorchToTcp.cpp

```diff
@@ -97,6 +94,11 @@ class ConvertTorchToTcp : public ConvertTorchToTcpBase<ConvertTorchToTcp> {
 
 } // namespace
 
+std::unique_ptr<OperationPass<func::FuncOp>> createConvertTorchToTcpPass() {
+  llvm::ArrayRef<std::string> emptyArrayRef;
+  return std::make_unique<ConvertTorchToTcp>(/*convertTorchOps=*/emptyArrayRef);
```

Comment:
Can this be `std::make_unique<ConvertTorchToTcp>({})`

Reply:
I initially had that, but hit this compiler error:
```
lib/Conversion/TorchToTcp/TorchToTcp.cpp:99:10: error: no matching function for call to 'make_unique'
  return std::make_unique<ConvertTorchToTcp>({});
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
/usr/lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/unique_ptr.h:961:5: note: candidate template ignored: substitution failure [with _Tp = mlir::tcp::(anonymous namespace)::ConvertTorchToTcp]: deduced incomplete pack <(no value)> for template parameter '_Args'
    make_unique(_Args&&... __args)                                                                                                                                                                     
    ^                                           
    ```